resources/bg.ftl

```diff
@@ -0,0 +1,100 @@
+-privacy_info = See WikiAuthBot2's [privacy statement](<https://wikiauthbot-ng.toolforge.org/ps>) to learn more about how we use your information.
+
+auth = За удостоверяване на автентичността на профила Ви в Уикипедия, моля, последвайте тази връзка: [Authenticate]({$url})
+
+    {-privacy_info}
+
+auth_exists_in_server = Вече сте удостоверили автентичността на профила си на този сървър. Няма нужда да се удостоверявате отново.
+
+auth_to_server = Вече сте идентифицирани като [{$name}](<{$url}>). Искате ли да удостоверите това на сървъра?
+
+    {-privacy_info}
+
+yes = Да
+no = Не
+
+auth_footer = Този линк ще бъде валиден в следващите 5 минути.
+
+authreq_canceled = Удостоверяването на автентичността е отменено.
+authreq_expired = Удостоверяването на автентичността е изтекло.
+
+authlog = {$mention} е удостоверен/а като [User:{$username}](<{$user_link}>) (id {$wmf_id})
```

Comment:
Do you also use "id"?

Reply:
> It would be the user's CentralAuth id.

Exactly, and IIRC it is indeed just an integer.